src/component/Restaurants.js

```diff
@@ -63,9 +93,45 @@ function Restaurants (props) {
                     </select>
                 </div>
             </form>
-
+            {restaurantList}
         </div>
     );
 }
 
+Restaurants.propTypes = {
+    selected: PropTypes.func.isRequired
+};
+
 export { Restaurants };
+
+const DEFAULT_SELECT_VALUE = "";
```

Comment:
what is gained by making this a constant?

Reply:
Clarifying that null is not an acceptable default value.

Reply:
Is that more clear this way, rather than just `useState("")`?

Reply:
changed